core/inc/knoting/skybox.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+#include <knoting/asset_manager.h>
+#include <knoting/shader_program.h>
+#include <knoting/texture.h>
+#include <knoting/types.h>
+#include <cereal/cereal.hpp>
+#include <memory>
+
+namespace knot {
+namespace components {
+
+// clang-format off
+
+enum class SkyBoxUniformSamplerHandle { SkyBox, Irradiance, Radiance, LAST };
+enum class SkyBoxTextureHandle { SkyBox, Irradiance, Radiance, LAST };
```

Comment:
Can we remove this now it's declared in types.h?

Reply:
these are for the handles of the shader and texture as they may not potentially match with the one in Types.h
as the ones in types.h are the public enums for graphics

Reply:
I removed the redundant enums for skybox, will consider similar cleanup in material.h when I am making renderer chnages